examples/full-jaffle-shop-demo/dbt/data/raw_orders.csv

```diff
@@ -98,3 +98,4 @@ id,user_id,order_date,status
 97,89,2018-04-07,placed
 98,41,2018-04-07,placed
 99,85,2018-04-09,placed
+100,85,null,placed
```

Comment:
Added for testing. I don't know if this will break some other test.

Reply:
Running e2e tests to make sure is doesn't